server/src/main/java/com/ahmadda/application/EventNotificationService.java

```diff
@@ -53,6 +72,38 @@ private void validateOrganizer(final Event event, final Long memberId) {
         }
     }
 
+    private List<OrganizationMember> resolveRecipients(
+            final Event event,
+            final List<Long> organizationMemberIds
+    ) {
+        Map<Long, OrganizationMember> organizationMembersById = getOrganizationMembersById(event);
+        validateOrganizationMemberIdsExist(organizationMembersById, organizationMemberIds);
+
+        return organizationMemberIds.stream()
+                .map(organizationMembersById::get)
+                .filter(Objects::nonNull)
+                .toList();
+    }
+
+    private Map<Long, OrganizationMember> getOrganizationMembersById(final Event event) {
+        return event.getOrganization()
+                .getOrganizationMembers()
+                .stream()
+                .collect(Collectors.toMap(OrganizationMember::getId, Function.identity()));
+    }
```

Comment:
p3
해당 메서드만 봤을때는 매게변수로 조직이 오는게 더 자연스러워 보이는데 어떻게 생각하시나용
```suggestion
    private Map<Long, OrganizationMember> getOrganizationMembersById(final Organization organization) {
        return organization
                .getOrganizationMembers()
                .stream()
                .collect(Collectors.toMap(OrganizationMember::getId, Function.identity()));
    }
```

Reply:
이견없습니다!!

[e9e5c9a](https://github.com/woowacourse-teams/2025-ah-madda/pull/255/commits/e9e5c9a8d3bf348d11d738acf07c002a736d81ee)에서 반영했습니다 ㅎㅎ~~